HealthAPILibrary/GiniHealthAPILibrary/Sources/GiniHealthAPILibrary/Documents/ExtractionResult.swift

```diff
@@ -20,7 +39,11 @@ import Foundation
     
     /// The line item compound extractions.
     public var lineItems: [[Extraction]]?
-        
+
+    public var isPayable: Bool {
```

Comment:
@razvancapra what is the benefit of having 2 interface points that are doing the same thing?
why do we need this variable?

Reply:
I've added this one so the client doesn't need to handle parsing on this item. But I can remove it and leave it totally on their side.